app/src/main/java/com/example/stromtracker/database/Geraete.kt

```diff
@@ -0,0 +1,137 @@
+package com.example.stromtracker.database
+
+
+import androidx.room.*;
+
+@Entity (indices = [ Index("kategorieID"), Index("haushaltID"), Index("raumID")  ], foreignKeys = [
+    ForeignKey(
+        entity = Kategorie::class,
+        parentColumns = arrayOf("kategorieID"),
+        childColumns = arrayOf("kategorieID"),
+        onDelete = ForeignKey.NO_ACTION),
+    ForeignKey(
+        entity = Haushalt::class,
+    parentColumns = arrayOf("haushaltID"),
+    childColumns = arrayOf("haushaltID"),
+        onDelete = ForeignKey.CASCADE
+    ),
+    ForeignKey(
+        entity = Raum::class,
+        parentColumns = arrayOf("raumID"),
+        childColumns = arrayOf("raumID"),
+        onDelete = ForeignKey.NO_ACTION
+)
+])
+
+    data class Geraete(
+        @ColumnInfo(name = "name") private var name: String,
+        @ColumnInfo(name = "kategorieID") private var kategorieID: Int,
+        @ColumnInfo(name = "raumID") private var raumID: Int,
+        @ColumnInfo(name = "haushaltID") private var haushaltID: Int,
+        @ColumnInfo(name = "stromVollast") private var stromVollast: Int,
+
+        @ColumnInfo(name = "stromStandBy") private var stromStandBy: Int,
+        @ColumnInfo(name = "betriebszeit") private var betriebszeit: Int,
```

Comment:
In dem Diagramm waren stromVollast, stromStandBy und betriebszeit als double geplant.
Ist das beabsichtigt, dass du int gewählt hast?
Ein Monitor beispielsweise benötigt ca. 0.8W im StandBy Verbrauch, was auf das Jahr gerechnet ca. 6kWh ausmacht.
Wählst du int um den Ganzjahresverbrauch zu speichern oder ist es sinnvoller den Verbrauch in kWh pro Tag zu speichern?

Reply:
Nein, das habe ich im Diagramm übersehen, kann man dann in Double umändern.